librz/core/cmd/cmd_zign.c

```diff
@@ -544,7 +545,7 @@ static int cmdFlirt(void *data, const char *input) {
 			eprintf("Usage: zfd filename\n");
 			return false;
 		}
-		rz_sign_flirt_dump(core->analysis, input + 2);
+		rz_core_flirt_dump(input + 2);
```

Comment:
In the future, we should split opening and parsing, and printing information from the parsed FLIRT file.

Reply:
agreed, but this is ok for now.